README.md

```diff
@@ -111,7 +118,8 @@ Or if you want to run every 20s while filtering out any "arriving" records
 Use Task Scheduler.
 
 #### cronic.py
-If the above sounds complicated, here's a simple looping script that calls `ttc_api_scraper.py` every minute during the TTC's operating hours. Just start it in your command line with  
```

Comment:
the green Master 2 got to be the present updated version, subject to items in transition.